packages/mandelbrot/assets/scss/components/_file-browser.scss

```diff
@@ -31,36 +18,37 @@
     font-weight: bold;
 }
 
-.select2-selection.select2-selection--single {
-    border-color: $color-frame-border;
-
-    .select2-container--focus & {
-        --border-color: rgb(var(--skin-links));
-
-        outline: none;
-        border-color: $color-link;
-        border-color: var(--border-color, $color-link);
-    }
-}
-
-.select2-results__option {
-    @include font(caption);
-
+.FileBrowser-select {
+    position: relative;
+    appearance: none;
     min-width: 15em;
-
-    &[aria-selected="true"] {
-        background-color: rgba($color-frame-border, 0.1) !important;
+    padding: 0 24px 0 8px;
```

Comment:
If you agree on making it higher, I would increase the side padding by 2px

Reply:
Gave it the exact same paddings as the Search input has, it should be fine like this.